lib/dom.js

```diff
@@ -3263,3 +3263,25 @@ declare var status: string;
 declare var top: WindowProxy;
 declare function getSelection(): Selection | null;
 declare var customElements: CustomElementRegistry;
+
+/* Notification */
+declare class Notification extends EventTarget {
```

Comment:
Looking at the spec, I think there are also these:

    static maxActions: number;
    static requestPermission(callback?: (perm: NotificationPermission) => mixed): Promise<NotificationPermission>;
    vibrate: Array<number>;
    renotify: boolean;

https://notifications.spec.whatwg.org/#api